prow/test/integration/test/deck_test.go

```diff
@@ -349,6 +349,24 @@ func TestRerun(t *testing.T) {
 	if err := updateJobConfig(context.Background(), kubeClient, rerunJobConfigFile, []byte(rerunJobConfig)); err != nil {
 		t.Fatalf("Failed update job config: %v", err)
 	}
+
+	// Now we are waiting on Horologium to create the first prow job so that we
+	// can rerun from.
+	// Horologium itself is pretty good at handling the configmap update, but
+	// not kubelet, accoriding to
+	// https://github.com/kubernetes/kubernetes/issues/30189 kubelet syncs
+	// configmap updates on existing pods every minute, which is a long wait.
+	// The proposed fix in the issue was updating the deployment, which imo
+	// should be better handled by just kicking off new pods.
+	// So here comes forcing restart of horologium pods.
+	if err := refreshProwPods(kubeClient, context.Background(), "horologium"); err != nil {
+		t.Fatalf("Failed kicking horologium: %v", err)
```

Comment:
Lets please be descriptive in our language, something like "failed to delete Horologium pods" or some such. Other than that, the change looks good to me

Reply:
good call, fixed